### PR TITLE
[TACHYON-445] Faster Data Server shutdown in unit tests

### PR DESCRIPTION
**Pull request by calvinjia**
https://tachyon.atlassian.net/projects/TACHYON/issues/TACHYON-445

Total test run time goes down by ~50% on my local machine with this change. Two new internal (not documented to users) settings are added which control the nature of the netty data server shutdown. The default values are the same as if `shutdownGracefully` is called without parameters.

### CODE REVIEW

**Comment by apc999** [comment]
LGTM. Thanks for the patch to improve integration test speed.

**Comment by haoyuan** [comment]
Awesome!

File: integration-tests/src/test/java/tachyon/master/LocalTachyonCluster.java [hunk 1]
```diff
@@ -207,7 +207,11 @@ public void start() throws IOException {
     mWorkerConf.set(Constants.WORKER_MIN_WORKER_THREADS, Integer.toString(1));
     mWorkerConf.set(Constants.WORKER_MAX_WORKER_THREADS, Integer.toString(100));
     mWorkerConf.set(Constants.WORKER_NETTY_WORKER_THREADS, Integer.toString(2));
-    
+
+    // Perform immediate shutdown of data server. Graceful shutdown is unnecessary and slow
+    mWorkerConf.set(Constants.WORKER_NETTY_SHUTDOWN_QUIET_PERIOD, Integer.toString(0));
+    mWorkerConf.set(Constants.WORKER_NETTY_SHUTDOWN_TIMEOUT, Integer.toString(0));
```
Comment (haoyuan) [inline integration-tests/src/test/java/tachyon/master/LocalTachyonCluster.java]: Could this be also applied to Multi-master cluster?

Comment (calvinjia) [inline integration-tests/src/test/java/tachyon/master/LocalTachyonCluster.java]: Good point, its best to be consistent.

**Pull request merged**